spec/interators/articles/save_spec.rb

```diff
@@ -0,0 +1,60 @@
+RSpec.describe Articles::Save do
+  subject(:call) { described_class.call(article: article, article_params: article_params) }
+
+  describe ".call" do
+    context "when creating a new article" do
+      let(:article) { build :article }
+      let(:article_params) { attributes_for :article }
+
+      it "succeeds and creates a new article" do
+        expect { call }.to change(Article, :count).by(1)
+        expect(call).to be_a_success
+      end
+
+      it "saves the article with correct attributes" do
+        call
+        expect(Article.last.title).to eq(article_params[:title])
+        expect(Article.last.body).to eq(article_params[:body])
```

Comment:
```suggestion
        expect(article.reload.title).to eq(article_params[:title])
        expect(article.reload.body).to eq(article_params[:body])
```

а вообще можно лучше
```
let(:expected_params) { { title: article_params[:title], body: article_params[:body]} }

it "saves the article with correct attributes" do
  call

  expect(article.reload).to have_attributes(expected_params)
end
```